src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -154,6 +154,13 @@ export class SummaryPageController extends PageController {
 
       viewModel.backLink = this.getBackLink(progress)
 
+      const missingEmailWarning =
+        await this.buildMissingEmailWarningModel(request)
+
+      if (missingEmailWarning) {
+        return h.view('summary', { ...viewModel, ...missingEmailWarning })
```

Comment:
we already have a `viewModel`. Why not include these attributes on that `viewModel` rather than creating a new view model containing other view models?

Reply:
I thought about it and the reason I didn't do it is - it didn't feel right to add form metadata to actual form model. Let me know what you think!

Reply:
DIscussed over a call. Resolved.